worq/core.py

```diff
@@ -138,7 +136,7 @@ def next_task(self, timeout=None):
             log.error('cannot deserialize task [%s:%s]',
                 self.name, task_id, exc_info=True)
             return None
-        #log.debug('next %s [%s:%s]', task.name, self.name, task_id)
+        # log.debug('next %s [%s:%s]', task.name, self.name, task_id)
```

Comment:
E265 block comment should start with '# '
I don't have time to update the PR, sorry.